pod/perlapio.pod

```diff
@@ -529,9 +529,9 @@ of passing NULL.)
 
 Portable cases are:
 
-    PerlIO_binmode(f,ptype,O_BINARY,NULL);
+    PerlIO_binmode(pTHX_ f,ptype,O_BINARY,NULL);
 and
-    PerlIO_binmode(f,ptype,O_TEXT,":crlf");
+    PerlIO_binmode(pTHX_ f,ptype,O_TEXT,":crlf");
```

Comment:
This should also use `aTHX_`, since these are examples of calling PerlIO_binmode(), and they would fail to compile with `pTHX_` on a threaded build.